src/Components/Ignitor/src/ElementHive.cs

```diff
@@ -42,39 +42,39 @@ public bool TryFindElementById(string id, [NotNullWhen(true)] out ElementNode? e
             foreach (var kvp in Components)
             {
                 var component = kvp.Value;
-                if (TryGetElementFromChildren(component, out element))
+                if (TryGetElementFromChildren(component, id, out element))
                 {
                     return true;
```

Comment:
This complains because `element` isn't guaranteed to be non-null. Only appears when building in VS